docs/developers/packaging.md

```diff
@@ -181,16 +185,16 @@ This means that if you install `napari=0.4.16` using the installer, the actual `
 can be found, by default, on the following locations:
 
 * Linux: `~/.local/napari-0.0.1/envs/napari-0.4.16/bin/napari`
-* MacOS: `~/Library/napari-0.0.1/envs/napari-0.4.16/bin/napari`
+* macOS: `~/Library/napari-0.0.1/envs/napari-0.4.16/bin/napari`
```

Comment:
This and above `napari=` could be kept up-to-date using the substitutions from #84 which hopefully will be merged shortly.
Just making a note.

Reply:
Nice! I'll wait :D

Reply:
It's been merged 😜

Reply:
so the substitution should be applied